main/src/ca/uwaterloo/flix/api/lsp/Position.scala

```diff
@@ -64,6 +65,9 @@ object Position {
   * @param line      Line position in a document (zero-based).
   * @param character Character offset on a line in a document (zero-based).
   */
-case class Position(line: Int, character: Int) {
+case class Position(line: Int, character: Int) extends Ordered[Position] {
   def toJSON: JValue = ("line" -> line) ~ ("character" -> character)
+
+  override def compare(that: Position): Int =
```

Comment:
I don't want a generic ordering on Position (because people could come to depend on it).

Instead I would just add a local max function to CodeActionProvider, if you need it.

Reply:
I don't believe it's needed anymore, but why not just leave it? I think it's the only logical way to order positions